src/pages/Admin/Charity/Dashboard/index.tsx

```diff
@@ -28,7 +27,6 @@ export default function Dashboard() {
       >
         {(proposals) => <Table proposals={proposals} />}
       </QueryLoader>
-      <Donations classes="mt-10" />
```

Comment:
contributions should also exists in dashboard
<img width="809" alt="image" src="https://user-images.githubusercontent.com/89639563/220275042-281602c2-236e-4d28-b297-54396b67528b.png">